packages/bsky/src/api/com/atproto/admin/getAccountInfos.ts

```diff
@@ -6,7 +6,7 @@ import { INVALID_HANDLE } from '@atproto/syntax'
 export default function (server: Server, ctx: AppContext) {
   server.com.atproto.admin.getAccountInfos({
     auth: ctx.authVerifier.optionalStandardOrRole,
-    handler: async ({ params, auth }) => {
+    handler: async ({ params, auth, req }) => {
```

Comment:
The lingering changes here can probably be removed.

Reply:
good eye!